apps/e2e/indicators-storybook-e2e/src/e2e/chevron.component.cy.ts

```diff
@@ -0,0 +1,22 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('indicators-storybook', () => {
```

Comment:
I took a look at our other implementations, and we add more context to the name, for example `forms-storybook - input box`. Should we add `- chevron`?